### DISCUSSION TITLE
Usage of `srcDir` option?

### DISCUSSION THREAD

**Discussion by danrocha** [vuejs/vitepress#661] [Q&A] [answered]
We would love to change the default `docs` folder name, but it seems not feasible at the moment, because that is the location Vitepress will always reach for the `/.vitepress` config folder, right? 

So, setting `srcDir: 'src'` in `/src/.vitepress/config.ts` has no effect, because Vitepress would not even find `/src/.vitepress/config.ts` in the first place, I assume.

However, even a solution in which I have `/docs/.vitepress/config.ts`, and my `.md` files in `/src`, while setting the `srcDir` in the config seems to have no effect at the moment (Vitepress never finds the .md files...)

Is this not yet implemented, or how would I go about doing this?

**Comment by kiaking** [comment, marked as answer]
No no, you specify the directly when executing the command.

```bash
$ vitepress dev         # srcDir is `./`

$ vitepress dev docs    # srcDir is `./docs`

$ vitepress dev oh-yeah # srcDir is `./oh-yeah`
```
